web/scripts/clean-css.ts

```diff
@@ -1,29 +1,45 @@
-// Remove unused colors from palette (as CSS Custom Properties)
-// and throw and error if other CSS Custom Properties are unused.
+// Applies PostCSS plugins to clean each CSS file generated during the build.
```

Comment:
Now the main description is too general. User need to read 10 lines to find *why* we need this script.

What do you think if we move description here:

```js
// 1. Merge all rules with :root selector into the single rule
// in the beginning of the file
// 2. Remove unused colors from palette (as CSS Custom Properties)
// and throw and error if other CSS Custom Properties are unused.
```

And then we can remove `PLUGINS` constant and use `let cssCleaner = postcss([rootsMerger, varsCleaner])`

Reply:
Great idea, I've updated the code